ui/jobs/jobs.css

```diff
@@ -635,26 +651,14 @@ div.justbuffs div#procs-container {
   background-color: rgb(133 0 0 / 70%);
 }
 
-.drk-color-bloodweapon.active {
-  background-color: rgb(199 0 0);
-}
-
-.drk-color-delirium {
+.drk-color-saltedearth {
   background-color: rgb(255 80 80 / 60%);
 }
 
-.drk-color-delirium.active {
-  background-color: rgb(255 80 80);
-}
-
 .drk-color-livingshadow {
   background-color: rgb(123 58 126 / 50%);
 }
 
-.drk-color-livingshadow.active {
-  background-color: rgb(179 0 189);
-}
```

Comment:
Is it good to remove these line in case that the Drk6xComponent might still using these?

Reply:
Drk6xComponent also not using these. It's 5.X heritage.

Reply:
Good